wrangler-core/src/main/java/io/cdap/wrangler/utils/ColumnConverter.java

```diff
@@ -284,4 +292,16 @@ private static Object convertType(String col, String toType, Object object) thro
         String.format("Column '%s' has value of type '%s' and cannot be converted to a '%s'.", col,
             object.getClass().getSimpleName(), toType));
   }
+
+  private static BigDecimal setDecimalScale(BigDecimal decimal, Integer scale, RoundingMode roundingMode)
+    throws DirectiveExecutionException {
+    if (scale == null) {
+      return decimal;
+    }
+    try {
+      return decimal.setScale(scale, roundingMode);
+    } catch (ArithmeticException e) {
+      throw new DirectiveExecutionException(e);
```

Comment:
Please add a proper error message similar to line 291

Reply:
Done